docs/BOOT_LOADER_SPECIFICATION.md

```diff
@@ -35,52 +35,57 @@ The target audience for this specification is:
 
 ## The boot partition
 
-Everything described below is located on a file system that will be called
-`$BOOT`. The boot loader or user-space programs reading the boot loader
-configuration should pick `$BOOT` according to the following rules:
+Everything described below is located on one or two partitions. The boot loader
+or user-space programs reading the boot loader configuration should locate them
+in the following manner:
 
 * On disks with an MBR partition table:
 
-  * A partition with the type ID of 0xEA shall be used as `$BOOT`.
+  * The boot partition — partition with the type ID of 0xEA — shall be used
+    for boot loader configuration and entries.
 
 * On disks with GPT (GUID Partition Table)
 
-  * If an Extended Boot Loader Partition (XBOOTLDR partition for short) — a
-    partition with GPT type GUID of `bc13c2ff-59e6-4262-a352-b275fd6f7172` —
-    is found, it should be used as `$BOOT`.
+  * The EFI System Partition (ESP for short) — a partition with GPT type GUID
+    of `c12a7328-f81f-11d2-ba4b-00a0c93ec93b` — should be used for boot loader
+    configuration and boot entries.
 
-  * Otherwise, the EFI System Partition (ESP for short) — a partition with
-    GPT type GUID of `c12a7328-f81f-11d2-ba4b-00a0c93ec93b` — should
-    be used as `$BOOT`.
+  * Optionally, an Extended Boot Loader Partition (XBOOTLDR partition for
```

Comment:
Just remembered: The XBOOTLDR partition must reside on the same disk where the ESP is located.

Reply:
Good points both. I updated the last commit.